features/search_iphone.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Search iphone
+
+Scenario: I want to search iphone case in amazon, and filtrer to price and polycarbonate
+#donde estoy
```

Comment:
when you automatize a project avoid comments in code. This is a training, you can comment on code.